formcentral/db.go

```diff
@@ -3,11 +3,21 @@ package main
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/jackc/pgx/v4/pgxpool"
 	"github.com/vlab-research/trans"
 )
 
+type Survey struct {
+   ID               string         `json:"id"`
+   Userid           string         `json:"userid"`
+   Form_json        trans.FormJson `json:"form_json"`
+   Shortcode        int            `json:"shortcode"`
+   Translation_conf trans.FormJson `json:"translation_conf"`
```

Comment:
Why is `translation_conf` of type `trans.FormJson`?

Reply:
Yeah, my bad. I thought I've read `Translation_FORM`. Checkout https://github.com/vlab-research/fly/pull/24/commits/d7fb45ae293e1be4a0a2fd3a80abeee24e190a18.

Let me know if we should define a `struct` for `Translation_conf`. So far, I've only found one reference to `translation_conf` potential `struct` here, https://github.com/vlab-research/fly/blob/e3adaf3445c36f1b2c3fb9c1fede87ce67b54b7d/dashboard-server/utils/surveys/survey.util.js#L8.